atk4/ui
# Add True Reload feature

in the future, we should fix all components and allow true reload for all of them

help welcomed

How to test:
1. open `/demos/javascript/reloading.php`
2. click `Reload but keep custom changes`
3. notice, that the input value stay unchanges
4. if you do not belive, change it manually again and click reload (step 3)
5. if you do not belive even more, open inspector, change the number after the input field -  and click reload (step 3)
